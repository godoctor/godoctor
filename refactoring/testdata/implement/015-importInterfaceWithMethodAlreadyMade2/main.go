package main // <<<<< stubInterface,1,1,1,1,os.FileInfo,FileInfoStruct,pass

import (
	"fmt"
	"os"
)

func main() {
	fmt.Println("works")
	os.Exit(0)
}

type Queue interface {
	Enqueue(x int)
	Dequeue() int
	ThreeQueue(x int) int
}

type Apple struct {

}

func (s *Apple) Sys() interface{} {
	return nil
}
