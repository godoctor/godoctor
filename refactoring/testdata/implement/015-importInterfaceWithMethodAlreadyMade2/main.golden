package main // <<<<< stubInterface,1,1,1,1,os.FileInfo,FileInfoStruct,pass

import (
	"fmt"
	"os"
)

func main() {
	fmt.Println("works")
	os.Exit(0)
}

type Queue interface {
	Enqueue(x int)
	Dequeue() int
	ThreeQueue(x int) int
}

type Apple struct {

}

func (s *Apple) Sys() interface{} {
	return nil
}

func (s *Apple) Size() int64 {
	return 0
}

func (s *Apple) Name() string {
	var eString string
	return eString
}

func (s *Apple) Mode() os.FileMode {
	return nil
}

func (s *Apple) ModTime() time.Time {
	return nil
}

func (s *Apple) IsDir() bool {
	return false
}
